src/components/AddImage.js

```diff
@@ -0,0 +1,39 @@
+import Component from './Component.js';
+
+class AddImage extends Component {
+
+    render() {
+        const form = this.renderDOM();
+        const onAdd = this.props.onAdd;
+
+        form.addEventListener('submit', event => {
+            event.preventDefault();
+
+            const formData = new FormData(form);
+
+            const newImage = {
+                title: formData.get('title'),
```

Comment:
On this image you are only ever adding a title and description but on your filter fucntion you filter the keyword too so you no long can filter after you add an item.

Ideally, you add another key box for key words but you can work around that by doing something like the following comment